aviary/examples/test/test_all_examples.py

```diff
@@ -11,12 +11,29 @@
 import unittest
 
 from openmdao.utils.testing_utils import use_tempdirs
+from openmdao.utils.general_utils import set_pyoptsparse_opt
 
 # TODO: Address any issue that requires a skip.
 SKIP_EXAMPLES = {
     'run_multimission_example_large_single_aisle.py': "Broken due to OpenMDAO changes",
 }
 
+OPT, OPTIMIZER = set_pyoptsparse_opt('SNOPT')
```

Comment:
Note, this is probably a temporary fix, until the error type is changed on openmdao: https://github.com/OpenMDAO/OpenMDAO/issues/3510